crates/net/network/src/transactions/fetcher.rs

```diff
@@ -57,6 +59,14 @@ pub(crate) struct TransactionFetcher {
 // === impl TransactionFetcher ===
 
 impl TransactionFetcher {
+    /// Sets up transaction fetcher with config
+    pub fn with_transaction_fetcher_config(&mut self, config: &TransactionFetcherConfig) {
+        self.info.soft_limit_byte_size_pooled_transactions_response =
+            config.soft_limit_byte_size_pooled_transactions_response;
+        self.info.soft_limit_byte_size_pooled_transactions_response_on_pack_request =
+            config.soft_limit_byte_size_pooled_transactions_response_on_pack_request;
+    }
+
```

Comment:
this type of function that takes a config as param should rather return `Self` instead of take a mutable reference to self. so it would be called instead of `TransactionFetcher::default()` in the `TransactionsManager` as I saw was happening now.